code/style.css

```diff
@@ -0,0 +1,314 @@
+/* ---------------------- GENERAL STYLING ---------------------- */ 
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&display=swap');
+
+* {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    -webkit-font-smoothing: antialiased;
+    text-rendering: optimizeLegibility;
+}
+
+html {
+    font-family: 'Roboto Mono', monospace;
+    background: #000;
+    color: #fff;
+}
+
+body {
+    padding: 0;
+    width: 70vw;
+    margin: auto;
+    text-shadow: 1px 1px #004D16;
+}
+
+h1 {
+    font-size: 40px;
+}
+
+h2 {
+    margin: 0 auto 30px auto;
+    padding: 0;
+    text-align: center;
+}
+
+p {
+    margin: 0;
+    padding: 0;
+}
+
+/* ---------------------- HEADER ----------------------*/ 
+header {
```

Comment:
You seem to style a lot of tags by selector name. We should avoid that, consider changing all of them to classes to keep consistency

Reply:
We thought it was unnecessary to add classes to a semantic tag if there was only one? We only have one header, and  in the header there is only one image and one p-tag. We would definitely add a class if there were multiple variants of the same selector/element. Is this ok? Also- aren't we supposed to use the semantic tags when possible? Is it just the styling selectors we're talking about here?

Reply:
Semantic tags are there mostly to address :
1.1 HTML structure (so any other developer can easly look for `<header />` tag in your code when searching for block of code usually presented on top of the page and usually containing navigation, logo and main website settings)
2. SEO (semantically done websites are ranked with better grade by google search engines and therefore appear in google suggestions on top).

What's most important for CSS however is consistency - if we decides to use classes as styling reference, all elements should obey this rule. It's a convention mostly because main assumption done during development is "I will probably change this code later on". It doesn't really matter if you will or not, what matters are boundaries that we impose on our code. If you would decide, let's say 6 months from now on, that you want to change some styles in header, there is just big possibility you will not search for reference already created in line 41 of your CSS file but rather create new one (it's just how it is, we as humans don't think about it all the time). That would lead to some overwriting of styles. Problem is that styles applied via selector and styles applied via class name have different importance. Id's have even different importance, and styling by attributes as well. And at this point there is chaos. And we can avoid all of that by reference to classes only.

Your projects until now are pretty small so it might sound too puristic to you, but believe me, if you will get used to bad practices now, it will bite you later on in less expected moment